cli/configure.js

```diff
@@ -12,11 +13,14 @@ let thumbnailChannelDefault = '@thumbnails';
 let thumbnailChannel = '';
 let thumbnailChannelId = '';
 
-const createConfigFile = (fileName, configObject) => {  // siteConfig.json , siteConfig
-  const fileLocation = Path.resolve(__dirname, `../site/config/${fileName}`);
+const createConfigFile = (fileName, configObject, topSecret) => {  // siteConfig.json , siteConfig
+  const fileLocation = topSecret
+    ? Path.resolve(__dirname, `../config/${fileName}`)
```

Comment:
When we spoke, I may have said `/config`, which I meant `/site/config` in context.
We don't need a separate directory, but if we use one I would suggest using `site/secrets`.

Reply:
Ok. What you said was literally true: The spee.ch repo's gitignore did indeed cover /config even though it didn't exist. Where would you like the gitignore for the site repo?